src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -54,23 +53,29 @@ public static void DownloadData_InvalidArguments_ThrowExceptions()
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null); });
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null, null); });
 
-            AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
+            if (!PlatformDetection.IsFullFramework) // netfx has bug and returns wrong argument name in exception.
+            {
+                AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
```

Comment:
Fair enough. I just suggested that to have a "cleaner" test where we don't do an `if (!FullFramework)` check and we just leave that to `AssertExtensions` by calling it with both expected parameters, but you are right, we shouldn't be testing buggy behavior on either side and we mostly care about .Net Core behavior being in place against Desktop. So being said that, this is good as is.